src/sentry/snuba/outcomes.py

```diff
@@ -0,0 +1,330 @@
+from typing import Any, Dict, List, Optional, Tuple
+from abc import ABC, abstractmethod
+from django.http import QueryDict
+from sentry.utils.snuba import raw_query
+from .dataset import Dataset
+from sentry_relay import DataCategory
+
+from sentry.snuba.sessions_v2 import (
```

Comment:
there is a potential to move some of this (and other funcs within other snuba modules) into a common that the endpoints can import / inherit from, but abstracting out didn't feel worth it here, so just importing from sessions directly.